app/assets/stylesheets/pin-index.css.scss

```diff
@@ -55,31 +58,38 @@ div.url{
 }
 
 a.link{
+  width: 100%;
   color: $primary-background-color;
-  padding-left: 5px;
+  padding-left: 10px;
 }
 
+.space{
```

Comment:
weird class name... why don't you put that rule directly in .url?

btw I think a proper way of using glyphicons would be to add the glyphicon class to a span, nested inside your a tag. So you would have ``` <a ...><span class="glyph......."></span><span class="url-text">your url here</span></a>```